spec/timeago_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'timeago'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.